web/src/beta/components/fields/DateTimeField/EditPanel/index.tsx

```diff
@@ -0,0 +1,116 @@
+import Button from "@reearth/beta/components/Button";
+import PanelCommon from "@reearth/beta/components/fields/CameraField/PanelCommon";
```

Comment:
Please move this in to fields/common if its not just a CameraField related thing anymore 🙏🏻